src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacExtensionDefaultAttribute/DacExtensionDefaultAttributeFix.cs

```diff
@@ -44,27 +44,60 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
 			if (attributeNode != null && (attributeNode.Name as IdentifierNameSyntax).Identifier.Text.Equals(_PXDefault))
 			{
 				bool isBoundField = IsBoundField(diagnostic);
-				string codeActionNameBound = (isBoundField)
-												? nameof(Resources.PX1030FixBound).GetLocalized().ToString()
-												: nameof(Resources.PX1030FixUnbound).GetLocalized().ToString();
+				string codeActionNameBound = nameof(Resources.PX1030FixBound).GetLocalized().ToString() ;
+				string codeActionNameUnbound = nameof(Resources.PX1030FixUnbound).GetLocalized().ToString();
 
 				CodeAction codeActionBound =
 					CodeAction.Create(codeActionNameBound,
-							cToken => ReplaceIncorrectDefaultAttribute(context.Document, context.Span, isBoundField, cToken),
+							cToken => AddToAttributePersistingCheckNothing(context.Document, context.Span, isBoundField, cToken),
```

Comment:
@saloponov why there is always codeaction registered? Should you register this codeaction in case of unbound fields?

Reply:
@SENya1990 Yes, 'CodeAction' must be registered for bound and unbound fields.

Reply:
@SENya1990 https://jira.acumatica.com/browse/ATR-216 contain one condition: 
"If a field is unbound and has PXDefault without PXPersistingCheck.Nothing, suggest two code fixes: set PersistingCheck to Nothing or replace PXDefault with PXUnboundDefault"

Reply:
@saloponov ok